### ISSUE TITLE
Created warning in log file if key/parameter "advanced" is used in the API

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-backend#242]
By merging of PR #233 and resolving of issue #236 the key/parameter "advanced" is now deprecated. The API should now create a warning in the log file if the key/parameter is used in the API. This is the second step towards removing it from the API.

When this issue has been resolved, a new ticket for removing the support from the backend should be created.

**Comment by pamasse** [comment]
Can be done once the validate layer is merged

**Comment by mattias-p** [comment]
Do you think there should be separate issues for the removal of each deprecated feature? Another way would be to annotate each deprecation notice with the version number of the initial deprecation, and adding an items to the release process for removal (and possibly announcements) of deprecations.

**Comment by pamasse** [comment]
It's not already done in the Input Validation Layout ?

**Comment by matsduf** [comment]
A new issue, #454, that requests its removal, has been opened.
